src/modules/components/member/memberDataListItem/memberDataListItem.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { Card, Heading, NumberFormat, Tag, formatterUtils } from '../../../../core';
+import { truncateEthAddress } from '../../../../utils/truncateEthereumAddress';
+import { MemberAvatar, type IMemberAvatarProps } from '../memberAvatar';
+
+export interface IMemberDataListProps extends Omit<IMemberAvatarProps, 'size' | 'responsiveSize'> {
```

Comment:
The data-list-item components should extend the `IDataListItemProps` interface, the data-list-item component will render an anchor tag by default, therefore we don't need a `handleClick` property